src/posit/connect/resources.py

```diff
@@ -43,3 +47,196 @@ def update(self, *args, **kwargs):
 class Resources:
     def __init__(self, params: ResourceParameters) -> None:
         self.params = params
+
+
+class Active(ABC, Resource):
+    def __init__(self, ctx: Context, parent: Optional["Active"] = None, **kwargs):
+        """A base class representing an active resource.
```

Comment:
a dumb question - what does "active" mean in this context? Are we naming it that as a callback to the Rails reference implementation? Or is this meaning something connect-specific?

Reply:
Yeah, it's just a callback to the Rails name. Open to suggestions if anyone has a better name idea.